debug.html

```diff
@@ -0,0 +1,270 @@
+    />
+    <meta
+      property="og:image"
+      content="https://raw.githubusercontent.com/Real-Dev-Squad/website-www/develop/img/Real-Dev-Squad-preview.png"
+    />
+
+    <!-- Twitter Meta Tag-->
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:site" content="@RealDevSquad" />
+    <meta property="twitter:title" content="Real Dev Squad" />
+    <meta
+      property="twitter:description"
+      content="Community of developers who collaborate to build Real software."
+    />
+    <meta
+      property="twitter:image"
+      content="https://raw.githubusercontent.com/Real-Dev-Squad/website-www/develop/img/Real-Dev-Squad-preview.png"
+    />
+
+    <script src="/js/analytics.js"></script>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-YRG12HRZKB"
+    ></script>
+
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag('js', new Date());
+
+      gtag('config', 'G-YRG12HRZKB');
+    </script>
+  </head>
+  <body>
+    <div class="dropdown hide">
+      <ul class="dropdown-list">
+        <li class="dropdown-item dropdown-link" id="myprofile-option">
+          My Profile
+        </li>
+        <hr class="line" />
+        <li class="dropdown-item dropdown-link" id="signout-option">
+          Sign out
+        </li>
+      </ul>
+    </div>
+    <nav>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <div class="nav-login">
+        <a
+          class="btn-login"
+          href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+        >
+          <button class="btn-login-text">
+            Sign In
+            <img src="img/github.png" alt="GitHub Icon" />
+          </button>
+        </a>
+        <div class="user-greet">
+          <a href="https://my.realdevsquad.com/">
+            <div class="user-greet-msg">Hello, User!</div>
+            <img class="user-profile-pic" />
+          </a>
+        </div>
+      </div>
+      <ul class="nav-menu">
+        <li class="nav-logo-li">
+          <a href="https://realdevsquad.com/">
+            <img src="img/Real-Dev-Squad@1x.png" alt="home nav logo" />
+          </a>
+        </li>
+        <li class="home-tab">
+          <a href="http://www.realdevsquad.com/">Home</a>
+        </li>
+        <li><a href="https://welcome.realdevsquad.com/">Welcome</a></li>
+        <li><a href="">Events</a></li>
+        <li><a href="https://members.realdevsquad.com/">Members</a></li>
+        <!-- <li><a href="https://crypto.realdevsquad.com/">Crypto</a></li> -->
+        <li><a href="https://status.realdevsquad.com/">Status</a></li>
+        <li><a class="active-tab" href="./debug.html">Debug</a></li>
```

Comment:
I think we shouldn't give this to it navbar because this is not a feature for everyone, it is only for people in discord.

Reply:
this just get showed in debug page only, user can only visit this page by visiting /debug